crates/plugins/src/install.rs

```diff
@@ -147,3 +233,170 @@ fn file_digest_string(path: impl AsRef<Path>) -> Result<String> {
     let digest_string = format!("{:x}", digest_value);
     Ok(digest_string)
 }
+
+fn do_fetch<'a>(
+    repo: &'a git2::Repository,
+    refs: &[&str],
+    remote: &'a mut git2::Remote,
+) -> Result<git2::AnnotatedCommit<'a>, git2::Error> {
+    let mut cb = git2::RemoteCallbacks::new();
+
+    // Print out our transfer progress.
+    cb.transfer_progress(|stats| {
+        if stats.received_objects() == stats.total_objects() {
+            print!(
```

Comment:
i think many of these prints should be tracing but we can do a cleanup later before PR if this is helpful for debugging